### PR TITLE
Add implementation for a single URLStreamHandler for resource: protocol

### PR DESCRIPTION
**Pull request by johanvos**

### CODE REVIEW

**Comment by johanvos** [comment]
This handles #1030 and obsoletes #1031

File: substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/JavaNetSubstitutions.java [hunk 1]
```diff
@@ -160,6 +171,34 @@ static URLStreamHandler getURLStreamHandler(String protocol) {
...
+                    }
+
+                    @Override
+                    public InputStream getInputStream() throws IOException {
+                        Resources.ResourcesSupport support = ImageSingletons.lookup(Resources.ResourcesSupport.class);
+                        // remove "protcol:" from url to get the resource name
+                        String resName = url.toString().substring(1+JavaNetSubstitutions.RESOURCE_PROTOCOL.length());
+                        final List<byte[]> bytes = support.resources.get(resName);
+                        if (bytes == null || bytes.size() < 1) {
+                            System.err.println("Couldn't find resource "+resName);
```
Comment (vjovanov) [inline substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/JavaNetSubstitutions.java]: Is this line required by the spec or the original implementation? I would like to get rid of it as it fails our checks for not using `System.err` here.

**Comment by johanvos** [comment]
Not required. I'll remove it. (I tend to be pretty verbose)

**Comment by cstancu** [comment]
The changes have been merged in https://github.com/oracle/graal/commit/56624f010fdf2acdb8a1f7e575b6d317e6c63103. Thanks for your contribution!

**Pull request closed without merging**